packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -131,12 +132,26 @@ export default function AppHeader({
             <Divider />
 
             <MenuItem
-              title={"התנתק"}
+              title={"התנתקות"}
               onClick={() => {
                 firebase.auth().signOut();
                 setShowSideBar(false);
               }}
-              icon={<ExitToAppIcon />}
+              icon={<FeatherLogOut/>}
+            />
+          </>
+        )}
+        {!loggedIn && (
```

Comment:
Instead of `!loggedId && ...` and `loggedId && ...` consider using a trinary condition (i.e `loggedIn ? ... : ... `)